x/pairing/keeper/unresponsive_provider.go

```diff
@@ -78,21 +77,13 @@ func (k Keeper) PunishUnresponsiveProviders(ctx sdk.Context, epochsNumToCheckCUF
 		}
 	}
 
-	ProviderChainID := func(provider, chainID string) string {
-		return provider + " " + chainID
-	}
-
 	// check all supported providers from all geolocations prior to making decisions
 	existingProviders := map[string]uint64{}
-	stakeEntries := map[string]epochstoragetypes.StakeEntry{}
```

Comment:
removing this map results in accessing the store again, will reduce performance

Reply:
done. see https://github.com/lavanet/lava/commit/cc17d46ad93f512ddaf8cc7372fb047d9edfbfbf